npm/src/globals.ts

```diff
@@ -1,15 +1,18 @@
-import { Rpc } from './registry';
+import { Rpc, Frontend } from './registry';
 import { AssetId } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/asset/v1/asset_pb';
 import { JsonGlobals } from './json';
 import { sha256Hash } from './utils/sha256';
 
 export class RegistryGlobals {
   readonly stakingAssetId: AssetId;
   readonly rpcs: Rpc[];
+  readonly websites: Frontend[];
+  /** @deprecated use `websites` instead */
   readonly frontends: string[];
```

Comment:
suggestion: we can just drop this field to guarantee folks don't use it after this package version

We can actually keep the same name (not frontends-v2) given the package hides the naming convention of the json it consumes.

Reply:
Good point! Since removing `frontends` is a breaking change, we might as well just change it to another type. Updated in the latest commit